extension/js/common/composer.ts

```diff
@@ -970,7 +978,7 @@ export class Composer {
         document.getElementById('input_text')!.focus(); // #input_text is in the template
         // Firefox will not always respond to initial automatic $input_text.blur()
         // Recipients may be left unrendered, as standard text, with a trailing comma
-        await this.composerContacts.parseRenderRecipients(this.S.cached('input_to_container')); // this will force firefox to render them on load
+        this.composerContacts.parseRenderRecipients(this.S.cached('input_to')).catch(Catch.reportErr); // this will force firefox to render them on load
```

Comment:
Any reason not to await this? Are there external API calls?

Reply:
Sorry, I think I added it accidentally because it could block the thread.

I tested it along with `await` and it works the same way, thanks!